<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Janela</class>
 <widget class="QWidget" name="Janela">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>498</width>
    <height>379</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Janela</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <widget class="QWidget" name="tab_preProcess">
      <attribute name="title">
       <string>Pré-processamento</string>
      </attribute>
      <widget class="QPushButton" name="pushButton_abrir">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>101</width>
         <height>21</height>
        </size>
       </property>
       <property name="font">
        <font>
         <italic>false</italic>
         <underline>false</underline>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>ArrowCursor</cursorShape>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Abrir arquivo...</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_relatorioGeral">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>241</width>
         <height>71</height>
        </rect>
       </property>
       <property name="title">
        <string>Relatório geral</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_relatorio">
          <property name="text">
           <string>  Relatório:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEdit_relatorio">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="font">
           <font>
            <stylestrategy>PreferDefault</stylestrategy>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>ArrowCursor</cursorShape>
          </property>
          <property name="text">
           <string>Nenhum</string>
          </property>
          <property name="frame">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_instancias">
          <property name="text">
           <string>Instâncias:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="label_atributos">
          <property name="text">
           <string>Atributos:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QLineEdit" name="lineEdit_atributos">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="cursor">
           <cursorShape>ArrowCursor</cursorShape>
          </property>
          <property name="text">
           <string>Nenhum</string>
          </property>
          <property name="frame">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEdit_instancias">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="cursor">
           <cursorShape>ArrowCursor</cursorShape>
          </property>
          <property name="text">
           <string>Nenhum</string>
          </property>
          <property name="frame">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox_atributos">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>241</width>
         <height>141</height>
        </rect>
       </property>
       <property name="title">
        <string>Atributos</string>
       </property>
       <widget class="QLineEdit" name="lineEdit_todosAtributos">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>191</width>
          <height>91</height>
         </rect>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
      <widget class="QLineEdit" name="lineEdit_5">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>10</y>
         <width>211</width>
         <height>321</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Tab 2</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <slots>
  <slot>abrirArquivo()</slot>
 </slots>
</ui>
